monitoring/remote_monitoring_demo/recommendations_infra_demo/recommendations_demo/recommendation_helper.sh

```diff
@@ -88,11 +88,15 @@ function run_monitoring_exp() {
 	RESULTS_FILE=$1
 	BULK_RESULTS=$2
 	DAYS_DATA=$3
+	EXP_TYPE=$4
+	if [ -z "${EXP_TYPE}" ]; then
+		EXP_TYPE="container"
+	fi
 	if [ -z "$DAYS_DATA" ]; then
-		echo "${SCRIPTS_REPO}/recommendation_experiment.py -c ${CLUSTER_TYPE} -p \"./recommendations_demo/json_files/resource_optimization_openshift.json\" -e \"./recommendations_demo/json_files/create_exp.json\" -r  ${RESULTS_FILE} -b ${BULK_RESULTS}"
-		python3 ${SCRIPTS_REPO}/recommendation_experiment.py -c ${CLUSTER_TYPE} -p "./recommendations_demo/json_files/resource_optimization_openshift.json" -e "./recommendations_demo/json_files/create_exp.json" -r  ${RESULTS_FILE} -b ${BULK_RESULTS}
+		echo "${SCRIPTS_REPO}/recommendation_experiment.py -c ${CLUSTER_TYPE} -p \"./autotune/manifests/autotune/performance-profiles/resource_optimization_openshift.json\" -e \"./recommendations_demo/json_files/create_exp.json\" -r  ${RESULTS_FILE} -b ${BULK_RESULTS} -t ${EXP_TYPE}"
```

Comment:
Assign these to variables and use the variables for readability. Also remove the if-else check